docs/spec/appendix.md

```diff
@@ -74,4 +74,7 @@ The following operation inputs are in the form of Sidetree REST API Operations.
 
 ```json
 [[insert: www/vectors/resolution/longFormResponseDidDocument.json]]
-```
\ No newline at end of file
+```
+
+### Acknowledgements
+Portions of the work on this specification have been funded by the United States Department of Homeland Security's (US DHS) Silicon Valley Innovation Program under contracts 70RSAT20T00000003, and 70RSAT20T00000033. The content of this specification does not necessarily reflect the position or the policy of the U.S. Government and no official endorsement should be inferred.
```

Comment:
I would suggest changing the language to: "ENTITY_NAME_HERE received funding from the United States Department of Homeland Security's (US DHS) Silicon Valley Innovation Program to contribute to this work item under contracts 70RSAT20T00000003, and 70RSAT20T00000033. This work item does not necessarily reflect the position or the policy of the U.S. Government and no official endorsement should be inferred."

I would have made an actual code change suggestion, but I am not aware of what entity/entities they are, so if someone can add that in comments I can make an actionable code suggestion.

Reply:
I'm ok with the suggested changes, but I won't implement them unless a formal process defines this for all WGs.

Reply:
@csuwildcat I suggest you implement those changes here: https://github.com/decentralized-identity/org/pull/9#issuecomment-1011317880

Reply:
@csuwildcat, good suggestion - better this way.